tests/cmdline/commands/test_group.py

```diff
@@ -261,6 +261,44 @@ def test_add_remove_nodes(self):
         self.assertClickResultNoException(result)
         self.assertEqual(group.count(), 0)
 
+    def test_move_nodes(self):
+        """Test `verdi group move-nodes` command."""
+        node_01 = orm.CalculationNode().store()
+        node_02 = orm.Int(1).store()
+        node_03 = orm.Bool(True).store()
+
+        # Add all three nodes to the first dummy group
+        result = self.cli_runner.invoke(
+            cmd_group.group_add_nodes, ['--force', '--group=dummygroup1', node_01.uuid, node_02.uuid, node_03.uuid]
+        )
+        self.assertClickResultNoException(result)
+
+        # Check if all nodes have been added successfully to the first dummy group
+        result = self.cli_runner.invoke(cmd_group.group_show, ['dummygroup1'])
+        self.assertClickResultNoException(result)
+        self.assertIn('CalculationNode', result.output)
+        self.assertIn(str(node_01.pk), result.output)
+        self.assertIn('Int', result.output)
+        self.assertIn(str(node_02.pk), result.output)
+        self.assertIn('Bool', result.output)
+        self.assertIn(str(node_03.pk), result.output)
+
+        # Move all three nodes to the second dummy group
```

Comment:
The test looks pretty neat in general, but would it perhaps be more versatile to only move two of the nodes? This way you are also checking the integrity of the nodes that are left on the source group.

Reply:
Good point! I should probably also add a test that checks if the command correctly fails in case we try to move nodes that are **not** in the `source_group`, once we fix this behaviour.